### ISSUE TITLE
Cannot read property '*' of undefined / using property of var inside includes

### ISSUE THREAD

**Issue by ingevlaar** [gulp-community/gulp-pug#132]
I use Pug in two ways. For development I run Express and Pug as view engine and for distribution I use gulp-pug. Works perfectly fine till today. 

I tried to run gulp-pug on a file which includes the following concept:

``` jade
- var partialData = [{title: "Partial 1"}, {title: "Partial 2"}];

each item in partialData
    span=item.title
    include ../partials/partial
```

and partials/partial.pug includes only:

``` jade
span=item.title
```

Works perfectly fine in development, but fails with gulp-pug

``` Batchfile
events.js:160
      throw er; // Unhandled 'error' event
      ^
 TypeError: C:\D-drive\_vsp\Boilerplate-Frontend\dev\html\components\partials\pa
rtial.pug:1
  > 1| span=item.title

Cannot read property 'title' of undefined
    at eval (eval at wrap (C:\D-drive\_vsp\Boilerplate-Frontend\dev\node_modules
\pug-runtime\wrap.js:6:10), <anonymous>:7:61)
    at template (eval at wrap (C:\D-drive\_vsp\Boilerplate-Frontend\dev\node_mod
ules\pug-runtime\wrap.js:6:10), <anonymous>:7:118)
    at DestroyableTransform.compilePug (C:\D-drive\_vsp\Boilerplate-Frontend\dev
\node_modules\gulp-pug\index.js:33:49)
    at DestroyableTransform.Transform._read (C:\D-drive\_vsp\Boilerplate-Fronten
d\dev\node_modules\readable-stream\lib\_stream_transform.js:159:10)
    at DestroyableTransform.Transform._write (C:\D-drive\_vsp\Boilerplate-Fronte
nd\dev\node_modules\readable-stream\lib\_stream_transform.js:147:83)
    at doWrite (C:\D-drive\_vsp\Boilerplate-Frontend\dev\node_modules\readable-s
tream\lib\_stream_writable.js:313:64)
    at writeOrBuffer (C:\D-drive\_vsp\Boilerplate-Frontend\dev\node_modules\read
able-stream\lib\_stream_writable.js:302:5)
    at DestroyableTransform.Writable.write (C:\D-drive\_vsp\Boilerplate-Frontend
\dev\node_modules\readable-stream\lib\_stream_writable.js:241:11)
    at write (C:\D-drive\_vsp\Boilerplate-Frontend\dev\node_modules\vinyl-fs\nod
e_modules\readable-stream\lib\_stream_readable.js:623:24)
    at flow (C:\D-drive\_vsp\Boilerplate-Frontend\dev\node_modules\vinyl-fs\node
_modules\readable-stream\lib\_stream_readable.js:632:7)
```

I hope someone can help me out.

**Comment by ingevlaar** [comment]
Found out what was wrong. Gulp tried to compile the partial.pug as a standalone file, but ofcourse it has no variables / locals set in that case. So it fails. 

I want to distribute also partials for our back-end party; so they can easily check and implement the small html components without struggeling into the plain html template.

I created a fallback with dummy data.

``` jade
if item === undefined
    - var item = {title: "Title"}
```

Works like a charm.
